src/coreclr/src/pal/src/exception/remote-unwind.cpp

```diff
@@ -1384,6 +1476,24 @@ StepWithCompactEncoding(const libunwindInfo* info, compact_unwind_encoding_t com
             
     }
     ERROR("Invalid encoding %08x\n", compactEncoding);
+#elif defined(TARGET_ARM64)
+    if (compactEncoding == 0)
+    {
+        TRACE("Compact unwind missing for %p\n", (void*)info->Context->Pc);
+        return false;
+    }
+    switch (compactEncoding & UNWIND_ARM64_MODE_MASK)
+    {
+        case UNWIND_ARM64_MODE_FRAME:
+            return StepWithCompactEncodingArm64Frame(info, compactEncoding);
+
+        case UNWIND_ARM64_MODE_FRAMELESS:
+            break;
```

Comment:
does this case need to be handled ?

Reply:
This is treated the same way as @mikem8361  handled the similar `amd64` case was handled above. I think it is reasonable.  The error message would probably be misleading in both cases though.

We are also skipping the `*_DWARF` cases in both targets.  They would have similar issues.

Reply:
It is possible both targets should simply return true here.  Or correct the SP and then return true.  @mikem8361 ?

Reply:
The `arm64` frameless case comments seem to imply there is a mechanism to save the non-volatile registers to the stack, but it is really unclear.  I can try to create a big function which hits this case and see what needs to be done.

Reply:
guess its fine to leave as is, perhaps just add a comment there for clarity?

Reply:
I chose to implement UNWIND_ARM64_MODE_FRAMELESS